onnxruntime/python/tools/transformers/fusion_attention.py

```diff
@@ -1133,6 +1145,13 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
                     return
         elif is_no_mask_attention:
             pass
+        elif has_rel_pos_bias:
+            mask_nodes = self.model.match_parent_path(
+                add_qk,
+                ["Mul", "Sub", "Cast", "Unsqueeze", "Unsqueeze"],
```

Comment:
Check also which axes used in unsqueeze? Does add_qk support such shape (looks like need broadcasting in kernel)?